README.md

```diff
@@ -1,4 +1,4 @@
-[![Build status](https://ci.appveyor.com/api/projects/status/v7w84n21w9yjspq7/branch/master?svg=true)](https://ci.appveyor.com/project/puppetlabs/pdk-planning/branch/master)
+[![Build status](https://ci.appveyor.com/api/projects/status/v7w84n21w9yjspq7/branch/main?svg=true)](https://ci.appveyor.com/project/puppetlabs/pdk-planning/branch/main)
```

Comment:
`https://ci.appveyor.com/project/puppetlabs/pdk-planning/branch/main` does not exist.

Reply:
Only just created the main branch, it should be created soon

Reply:
that should appear when appveyor runs